monitoring/snippets/v3/uptime-check-client/snippets_test.py

```diff
@@ -75,31 +74,26 @@ def test_update_uptime_config(capsys):
         # in this case.
         @backoff.on_exception(backoff.expo, DeadlineExceeded, max_time=120)
         def call_sample():
-            snippets.update_uptime_check_config(
-                fixture.config_get.name, new_display_name, new_uptime_check_path)
+            return snippets.update_uptime_check_config(
+                fixture.config_get.name, new_display_name, new_uptime_check_path
+            )
 
-        call_sample()
+        result = call_sample()
 
-        out, _ = capsys.readouterr()
-        snippets.get_uptime_check_config(fixture.config_get.name)
-        out, _ = capsys.readouterr()
-        assert new_display_name in out
-        assert new_uptime_check_path in out
+        assert new_display_name == result.display_name
+        assert new_uptime_check_path == result.http_check.path
 
 
-def test_get_uptime_check_config(capsys, uptime):
-    snippets.get_uptime_check_config(uptime.config_get.name)
-    out, _ = capsys.readouterr()
-    assert uptime.config_get.display_name in out
+def test_get_uptime_check_config(capsys, uptime) -> None:
+    config = snippets.get_uptime_check_config(uptime.config_get.name)
+    assert uptime.config_get.display_name == config.display_name
 
 
-def test_list_uptime_check_configs(capsys, uptime):
-    snippets.list_uptime_check_configs(uptime.project_name)
-    out, _ = capsys.readouterr()
-    assert uptime.config_get.display_name in out
+def test_list_uptime_check_configs(capsys, uptime) -> None:
```

Comment:
`capsys` most probably is no longer needed

Reply:
done